### PR TITLE
Display built-in functions first in SHOW FUNCTIONS results

### PR DESCRIPTION
**Pull request by caithagoras**
In SHOW FUNCTIONS results, list the built-in functions first, and then
the SQL functions, in alphabetical order of the qualified function
names.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-parser/src/main/java/com/facebook/presto/sql/QueryUtil.java [hunk 1]
```diff
@@ -112,6 +112,11 @@ public static SortItem ascending(String name)
         return new SortItem(identifier(name), SortItem.Ordering.ASCENDING, SortItem.NullOrdering.UNDEFINED);
     }
 
+    public static SortItem descending(String name)
+    {
+        return new SortItem(identifier(name), SortItem.Ordering.ASCENDING, SortItem.NullOrdering.UNDEFINED);
```
Comment (rongrong) [inline presto-parser/src/main/java/com/facebook/presto/sql/QueryUtil.java]: This says `ASCENDING`. I'm very confused. >_< Could you add a test?

**Comment by caithagoras** [comment]
@rongrong 
Comments addressed. Test added.

**Comment by caithagoras** [comment]
@rongrong Tests added - supporting testing SQL functions in unit tests. PR is ready for review.

**Comment by caithagoras** [comment]
[test facebook]

**Pull request merged**